wp-content/themes/great-outdoor/header.php

```diff
@@ -39,7 +39,9 @@
 
         <header class="header">
 
-            <a href="<?php echo esc_url( bloginfo('url') ); ?>"><h1><?php esc_html( bloginfo('name') ); ?></h1></a>
+            <a href="<?php echo esc_url( bloginfo('url') ); ?>">
+                <img src="<?php echo get_stylesheet_directory_uri() . '/assets/logo.png' ?>" alt="GOPC logo">
```

Comment:
I'm wondering if we should use `.logo` or some other class here to add specificity to this particular image.  I can think of a few places in the mega-menus where images are being used, and `header img` would likely cascade down. Alternatively perhaps we just style it with `header > img`?